src/schema/v2/me/collectArtistsMutation.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { uniq } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { UserInterest, userInterestType } from "../userInterests"
+
+interface Input {
+  artistIDs: string[]
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "collectArtistsSuccess",
+  fields: () => ({
+    userInterests: {
+      type: new GraphQLNonNull(new GraphQLList(userInterestType)),
+      resolve: (userInterests) => userInterests,
+    },
+  }),
+})
+
+const FailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "collectArtistsFailure",
+  fields: () => ({
+    mutationError: {
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
This shouldn't be a string - there's a Gravity mutation error type which includes a richer schema, it should be something like https://github.com/artsy/metaphysics/blob/e1d3950fc75966e218d53660633918452c7cae2f/src/schema/v2/Feature/CreateFeatureMutation.ts#L54.